Scenarios/Project Apollo - NASSP/Apollo - Mission Scenarios/Apollo 10/Apollo 10 - 12 - Before Phasing TIG-6min.scn

```diff
@@ -670,6 +670,22 @@ AGC_BEGIN
   EMEM1447 76067
   EMEM1450 2745
   EMEM1451 56063
+  EMEM1452 74157
+  EMEM1453 76270
+  EMEM1454 77011
+  EMEM1455 74166
+  EMEM1456 77231
+  EMEM1457 76540
+  EMEM1460 63
+  EMEM1461 77532
+  EMEM1462 231
+  EMEM1463 23
+  EMEM1464 367
+  EMEM1465 321
+  EMEM1466 276
+  EMEM1467 71
+  EMEM1470 205
+  EMEM1470 77745
```

Comment:
Copy and paste mistake, second EMEM1470 should be removed (it comes from the LGC erasable).